src/System.Net.Security/src/System/Net/Security/SslState.cs

```diff
@@ -1449,6 +1420,33 @@ private void FinishHandshake(Exception e, AsyncProtocolRequest asyncRequest)
             }
         }
 
+        private void ProcessPendingWrite()
+        {
```

Comment:
Do we require being called while holding the lock?  If so, that'd be good to assert here.

Reply:
I added an assert, but it didn't register it in Github as changed for some reason, maybe the line numbers moved too much.